bin/pycbc_optimal_snr

```diff
@@ -210,7 +211,7 @@ if __name__ == '__main__':
 
     logging.info("Loading injections")
     injections = pycbc.inject.InjectionSet(opts.inj_xml)
-    inj_table = sorted(injections.table, key=lambda x: x.geocent_end_time)
+    inj_table = sorted(injections.table, key=lambda x: random.random())
```

Comment:
I don't this is reproduceable. The standard random seems to not give me consistent results on rerunning. If you replace with the numpy version, I think we can expect to get the same results each time.

Reply:
Why does it need to be consistent here? The table is sorted again before outputting.

Reply:
Well, it means that if a problem comes up, we might not be able to reproduce the conditions exactly, which can be frustrating..

Reply:
Swapped to numpy anyway